dot/rpc/modules/state.go

```diff
@@ -98,7 +98,9 @@ type StateCallResponse struct {
 type StateKeysResponse [][]byte
 
 // StateStorageDataResponse field to store data response
-type StateStorageDataResponse string
+type StateStorageDataResponse struct {
+	StorageData string
+}
```

Comment:
same here, will this change the json formatting? will it still match the spec?

Reply:
This wouldn't work and it caused the regression reported in #1231. I have fixed it.